packages/ui/src/views/unified-pipeline-studio/components/unified-popover-commit-info.tsx

```diff
@@ -19,10 +19,7 @@ export const PopoverCommitInfo = {
     return (
       <div className="flex items-center justify-between p-3">
         <div className="flex items-center gap-x-3">
-          <Avatar.Root size="8" className="inline-flex">
-            {!!avatarUrl && <Avatar.Image src={avatarUrl} />}
-            <Avatar.Fallback>{initials}</Avatar.Fallback>
-          </Avatar.Root>
+          <Avatar name={initials} src={avatarUrl} size="lg" rounded />
```

Comment:
Looks like we need to swap this to `authorName` and remove `initials` from this component.
```suggestion
          <Avatar name={authorName} src={avatarUrl} size="lg" rounded />
```

Reply:
Done